src/io/flutter/FlutterInitializer.java

```diff
@@ -132,6 +133,9 @@ public void runActivity(@NotNull Project project) {
 
     FlutterRunNotifications.init(project);
 
+    // Watch save actions.
+    FlutterReloadManager.getInstance(project);
```

Comment:
It's not obvious to me what's happening here, I'm guessing there's a side effect?  Maybe update the comment to make it more obvious what's going on?